README.md

```diff
@@ -23,16 +23,34 @@ are running the simulation from.
 
 ## Running
 
+### Run with python virtual environment
+
 With the simulation package installed, you should have a new command-line
-program called `simulation ` available.  Try running `simulation --help` to get
+program called `nlisim ` available.  Try running `nlisim --help` to get
 more information.  To run a simulation, you will need configure a configuration.
 There is an example configuration in the repository to get you started.
 
-Now run simulation up to 50 seconds using the first example config.
+Now run simulation up to 50 timesteps using the first example config.
 ```bash
     nlisim --config config.ini.example run 50
 ```
 
+### Run with docker container
+
+Build the docker image.
+```bash
+    docker build -t nlisim .
+```
+
+Run the container with imported configuration file and geometry file up to 50 timesteps. Configurate the output path.
+```bash
+    docker run \
+        -v $(pwd)/config.ini:/opt/nlisim/config.ini \
```

Comment:
This file will not exist for most people. It should be `$(pwd)/config.ini.example`.

Reply:
Worse, in the case the file doesn't exist, using `-v` with [Docker will cause the non-existent path to be automatically created as a directory on the host system](https://github.com/moby/moby/issues/13121), which is the sort of thing that will ruin someone's afternoon as they attempt to debug it. I think we should switch to the more verbose `--mount` syntax, which does not have the same behavior or potential to cause confusion.